src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -39,36 +48,67 @@ export class HuiGlanceCardEditor extends hassLocalizeLitMixin(LitElement)
         label="Title"
         value="${this._config!.title}"
         .configValue="${"title"}"
+        .type="${"input"}"
         @value-changed="${this._valueChanged}"
-      ></paper-input
+      ></paper-input>
+      <hui-theme-select-editor
+        .hass="${this.hass}"
+        .value="${this._config!.theme}"
+        .type="${"input"}"
+        .configValue="${"theme"}"
```

Comment:
Sometimes I wonder if we're going too far with this. We could also just make a `this._themeChanged` function.

I am not sure what I prefer.

Reply:
I do know that if we would ever get TypeScript type checking properties set on HTML tags in HTML templates (the data is there, it's just not supported yet), this would be a thing that we would have to change.

Reply:
We would need the .configValue for the inputs or some other way to know what value they would be changing, right?